public/css/viewEventsStyle.css

```diff
@@ -0,0 +1,155 @@
+
+body {
+margin: 0px;
+padding: 0px;
+height: 100vh;
+width: 100%;
+}
+
+.container {
+width: 90%;
+display: flex;
+align-self: center;
+justify-content: space-between;
+flex-wrap:wrap;
+
+}
+
+
+.bigContainer {
+display: flex;
+/* height: 200px; */
+flex-direction: column;
+flex-wrap: wrap;
+}
+
+
+
+/* .input {
+background-color: #742223;
+border: 0;
+color: white;
+font-family: Roboto;
+font-style: Regular;
+font-size: 15px;
+} */
+
+.input:hover {
+font-size: 17px;
+color: #eae4ac;
+cursor: pointer;
+}
+
+.boxs {
+display: flex;
+margin-top: 25px;
+margin-left: 25px;
+margin-right: : 25px;
+justify-content: flex-start;
+flex-wrap: wrap;
+}
+
+.box {
+display: flex;
+width: 1000px;
+margin: 0 10px;
+flex-direction: row;
+align-items: center;
+flex: 1;
+background-color: #828282;
+margin-bottom: 25px;
+margin-left: 90px;
+}
+
+.imgE{
+width: 250px;
+  height: 250px;
+}
+
+.innerbox {
+display: flex;
+border-radius: 50%;
+align-items: center;
+justify-content: center;
+width: 260px;
+height: 260px;
+border: 1px solid #828282;
+}
+
+.innerboxinfo{
+display: flex;
+flex-wrap: wrap;
+flex-direction: column;
+/* margin-right: 90px; */
+/* border-radius: 50%; */
+align-items: center;
+justify-content: space-around;
+width: 260px;
+height: 260px;
+border: 1px solid #828282;
+}
+
+.a {
```

Comment:
Describe your classnames better.

Reply:
@NoureldeanSaed  actually i will push the pages style in a separated branch, so i'll change the classnames :+1: